### PR TITLE
RHIDP-428 Add JUnit test results reporter

### PR DESCRIPTION
**Pull request by zdrapela**
## Description

Add a JUnit reporter for Playwright e2e tests.

## Which issue(s) does this PR fix

- Prerequisite for Logilica and ReportPortal onboarding.

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,9 +50,8 @@ site
 
 .turbo
 
-# Cypress
-**/cypress/downloads
-**/cypress/screenshots
+# e2e test results
+e2e-tests/junit-results.xml
```
Comment (subhashkhileri) [inline .gitignore:54]: you may remove from root `.gitignore`
```
# Cypress
**/cypress/downloads
**/cypress/screenshots
```

but better we move  `junit-results.xml` to `e2e-tests/.gitignore`

**Review by subhashkhileri — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (zdrapela) [reply .gitignore]: Yes, that makes sense. Thank you

**Pull request merged**